pkg/config/genesis_extensions.go

```diff
@@ -1,32 +1,61 @@
 package config
 
 import (
+	"encoding/base64"
 	"fmt"
 
 	"github.com/nspcc-dev/neo-go/pkg/core/native/noderoles"
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
 // GenesisExtensions represents the extensions enabled in the genesis block or by the
 // moment of native contracts initialization.
 type GenesisExtensions struct {
 	// Roles contains the set of roles that should be designated on native
 	// Designation contract initialization.
-	Roles map[noderoles.Role]keys.PublicKeys
+	Roles        map[noderoles.Role]keys.PublicKeys
+	Transactions []GenesisTransaction
```

Comment:
And this probably can be a single transaction, a script can do anything.

Reply:
Fixed.